pkg/deployment/resources/pod_creator.go

```diff
@@ -379,6 +390,21 @@ func (r *Resources) RenderPodForMember(ctx context.Context, acs sutil.ACS, spec
 			memberStatus: m,
 			cachedStatus: cache,
 		}
+	case api.ServerGroupTypeGateway:
+		imageInfo.Image = r.context.GetOperatorImage()
```

Comment:
Is it possible to get .spec.gateway.image ? IF no, please create ticket for that

Reply:
Ticket created